apps/web/src/views/Launchpad/CreateCampaign.tsx

```diff
@@ -39,32 +39,32 @@ export const CreateCampaign: React.FC = () => {
     window.scrollTo(0, 0)
   }
 
-  const getPaid = async () => {
+  const getKyced = async () => {
     const response = await fetch(`/api/kyc-info/${address}`)
     const data = await response.json()
 
     if (data.status !== 'REJECTED') {
```

Comment:
This check looks wrong, I think it should be for "MINTED". Otherwise all other states like pending would be recognized as KYCed.